application/src/test/java/org/opentripplanner/service/paging/TestPagingUtils.java

```diff
@@ -13,7 +13,8 @@ static String cleanStr(Object value) {
       .replaceAll("2020-02-01T(\\d\\d:\\d\\d):00Z", "$1-1d")
       .replaceAll("2020-02-02T(\\d\\d:\\d\\d):00Z", "$1")
       .replaceAll("2020-02-03T(\\d\\d:\\d\\d):00Z", "$1+1d")
-      .replaceAll("0(\\d:00)", "$1");
+      .replaceAll("0(\\d:00)", "$1")
+      .replaceAll("0(\\d:01)", "$1");
```

Comment:
This class could also use some comments javadoc and maybe even comments to explain what each of these replaces do.